src/locales/de_DE/de_DE.json

```diff
@@ -476,9 +476,11 @@
         "title": "Lassen Sie uns sicherstellen, dass sich niemand für Sie ausgibt"
     },
     "welcome": {
-        "description_p_1": "Um ein Bankkonto zu eröffnen, müssen wir Ihre Identität überprüfen.",
-        "description_p_2": "Es wird nur ein paar Minuten dauern.",
         "next_button": "Identität überprüfen",
-        "title": "Eröffnen Sie Ihr neues Bankkonto"
+        "title": "Identität überprüfen",
+        "subtitle": "It should take a few minutes",
+        "description_p_1": "Use your webcam or phone to photograph:",
+        "description_p_2": "1. your identity document",
+        "description_p_3": "2. your face"
```

Comment:
Why is this partially translated? Same for the other languages

Reply:
I can update them with the latest from Lokalise now, but the description keys all have the localised translations for "Verify Identity". Have raised this question in Slack about the translations.

Reply:
Updated to use the latest key-string for default descriptions after discussion with Rae.